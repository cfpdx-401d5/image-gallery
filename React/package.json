{
  "name": "image-gallery",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "body-parser": "^1.17.1",
    "cors": "^2.8.1",
    "express": "^4.15.2",
    "mongoose": "^4.8.7",
    "morgan": "^1.8.1",
    "react": "^15.4.2",
    "react-dom": "^15.4.2"
  },
  "devDependencies": {
    "chai": "^3.5.0",
    "chai-http": "^3.0.0",
    "enzyme": "^2.7.1",
    "enzyme-to-json": "^1.5.0",
    "eslint": "^3.17.1",
    "eslint-plugin-react": "^6.10.0",
    "mocha": "^3.2.0",
    "nodemon": "^1.11.0",
    "react-addons-test-utils": "^15.4.2",
    "react-scripts": "0.9.4"
  },
  "scripts": {
    "start": "react-scripts start",
    "start-server": "nodemon server.js",
    "build": "react-scripts build",
    "test": "react-scripts test --env=jsdom",
    "eject": "react-scripts eject"
  }
}
