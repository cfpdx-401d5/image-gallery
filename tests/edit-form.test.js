/* eslint-disable */
import React from 'react';
import {shallow, render} from 'enzyme';
import toJson from 'enzyme-to-json';
import EditForm from '../src/forms/EditForm';

describe('list display', () => {
    it('list display', () => {
        const rendered = shallow(<EditForm item={{}} onChange={() => {}} onDisplay={() => {}} onEdit={() => {}} />);
        expect(toJson(rendered)).toMatchSnapshot();
    });
});