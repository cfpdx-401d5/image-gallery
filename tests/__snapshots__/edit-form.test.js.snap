exports[`list display list display 1`] = `
<form>
  <fieldset>
    <legend>
      Edit Image Information: 
    </legend>
    <TextField
      controlFunc={[Function]}
      inputType="text"
      name="title"
      title="Title: " />
    <TextArea
      controlFunc={[Function]}
      name="description"
      resize={false}
      rows={5}
      title="Description: " />
    <TextField
      controlFunc={[Function]}
      inputType="text"
      name="url"
      title="URL: " />
    <button
      className="buttons"
      onClick={[Function]}>
      Submit
    </button>
    <button
      className="buttons"
      onClick={[Function]}>
      Cancel
    </button>
  </fieldset>
</form>
`;
